src/seedu/addressbook/AddressBook.java

```diff
@@ -816,6 +860,38 @@ private static void addPersonToAddressBook(HashMap<PersonProperty, String> perso
         ALL_PERSONS.add(person);
         savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
     }
+    
+    /**
+     * Edits a person in the address book. Saves changes to storage file.
+     *
+     * @param updated person to edit
+     */
+    private static void editPersonToAddressBook(HashMap<PersonProperty, String> person) {
+    	if (updatePersonChanges(person)) {
+    		savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
```

Comment:
indentation is off here.
we require the code to be indented with 4 spaces (not tabs)
Tip: You can get eclipse to do this for you; check out how?